server/routes/superheroes_routes.js

```diff
@@ -0,0 +1,91 @@
+const superheroesRouter = require('express').Router();
+const { Superhero } = require('../db/models/models_index.js');
+
+// app.use('/api/superheroes', superheroesRouter) in routes_index.js
+
+// get all superheroes
+superheroesRouter.get('/', async (req, res) => {
+    
+    try {
+        const superheroes = await Superhero.findAll();
+        res.status(200).send({ superheroes });
+    } catch(e) {
+        console.log(e);
+        res.status(400).send({ message: 'Superheroes not found' });
```

Comment:
I think it should be 404 if not found or keep 400, but the message should be something like Server Error